src/main/java/com/apt/wii/web/rest/ContentResource.java

```diff
@@ -154,6 +154,23 @@ public ContentResource(ContentService contentService, ContentRepository contentR
         return ResponseUtil.wrapOrNotFound(contentDTO);
     }
 
+    /**
+     * {@code GET  /contents/:id} : get the "id" content.
+     *
+     * @param id the id of the contentDTO to retrieve.
+     * @return the {@link ResponseEntity} with status {@code 200 (OK)} and with body the contentDTO, or with status {@code 404 (Not Found)}.
+     */
+    @GetMapping("/question/{id}/contents")
+    public List<ContentDTO> getContentByQuestion(
+        @PathVariable Long id,
+        @RequestParam(defaultValue = "0") Integer pageNo,
+        @RequestParam(defaultValue = "10") Integer pageSize
+    ) {
```

Comment:
Why Pagination require here ? shouldn't we fetch all the contents of a question ?